app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -62,6 +62,17 @@ $(function() {
           width: 'resolve',
           allowClear: allowClear
         });
+
+        var $parent = $(el).parent(), $label = $parent.find('label');
+
+        if ($label.length) {
+          var $input = $parent.find('#' + $label.attr('for'))
+
+          $(el).on('change', function(value) {
```

Comment:
'$' is not defined                                                 no-undef<br>Missing space before function parentheses                          space-before-function-paren<br>'value' is defined but never used                                  no-unused-vars